etc/settings.yaml.in

```diff
@@ -1,25 +1,66 @@
 locations:
-  input-dir: in/
-  output-dir: out/
-  genome-dir: genome/
+  input-dir: /path/to/reads/
+  output-dir: /path/to/output/
+  genome-fasta: /path/to/genome.fasta
 
 general:
-  assembly: hg19
+  assembly: ''
+  # use one or both bisulfite aligners
+  # bwameth is faster and uses less resources
+  use_bwameth: False
+  # bismark is gold standard and uses sensible defaults
+  use_bismark: True
```

Comment:
This is nitpicking, but instead of use_bwameth and use_bismark, which allows for both to be set to True and both to be set to False, I'd use "aligner: bwameth | bismark".  This ensures at the config level that only one aligner is enabled and that it is one of the valid choices.